ArduPlane/Parameters.cpp

```diff
@@ -1086,6 +1086,7 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Bitmask: 11: Disable suppression of fixed wing rate gains in ground mode
     // @Bitmask: 12: Enable FBWB style loiter altitude control
     // @Bitmask: 13: Indicate takeoff waiting for neutral rudder with flight control surfaces
+    // @Bitmask: 14: In AUTO, climb to next waypoint altitude immediately instead of linear climb
```

Comment:
```suggestion
    // @Bitmask: 14: In AUTO - climb to next waypoint altitude immediately instead of linear climb
```
Can't have commas here.

commas separate bitmask entries, so you can have `Bitmask: 5:Bob, 6:Mary` - a syntax we use elsewhere.  Splitting the bitmask across lines kind of confuses things...